meetings/2023-12-06.md

```diff
@@ -0,0 +1,156 @@
+    * (screen share)
+    * working with INRIA and startin blox
+    * multiple pods (32) with 40 users
+        * each use with skills: Java, PHP, Python
+        * each lives in a city
+    * App will query based on skill and city
+    * Two scenarios for indexing
+        * Global index (federation pod): the indexes of the whole federation are merged on one pod (the federation's one).
+        * Distributed index (each pod): each pod maintains its own index
+    * Interface for entering skills and/or city. User can choose indexing strategy.
+    * Federated query involves many more HTTP requests and can be slower
+    * Using a direct pod traversal (per user) and is much slower
+    * The examples use public data, there are some interesting use cases to explore with private data
+    * PA: each pod is on a different docker container, can control the indexing strategy and explore the tradeoffs related to performance
+    * The project is available at https://github.com/Wimmics/solid-start/tree/main/sib-replica
+    * There should be more demos coming
+    * eP: one potential real world candidate for this sort of indexing relates to bike commuting
+
+### Holidays break in December
+URL: https://www.w3.org/groups/cg/solid/calendar/
+
+* eP: Last 2023 CG call on Dec 13th
+* eP: First 2024 CG call on Jan 10th
+
+### Special Topic Meetings
+URL: https://github.com/solid/specification/discussions/555
+
+* eP: We should schedule or cancel the Jan 9th meeting
+    * This should be decided on the 13 Dec CG call
+* eP: We can still schedule something on Dec 12th (currently canceled)
+* eP: Joint meeting with Social CG shows interest in future focused meetings. For example AuthN/AuthZ, data collections/containers etc.
+
+ACTION: eP will ask Sarven about scheduling the Dec 12 meeting to discuss launcher apps
+
+### Solid CG Chair Election Procedure
+URL: https://github.com/solid/specification/discussions/582
+
+* eP: One more week to vote: https://www.w3.org/2002/09/wbs/110151/cfb1563d-40eb-4c65-9859-30ee63e739bb/
+* eP: Pleaes double check if your name appears on https://github.com/w3c-cg/solid/blob/main/elections/2023/eligible-voters.txt
+    * If your name is not listed, please raise an issue or make a PR
+* eP: Any CG member can use the online poll, only votes cast by names listed above will be counted! We plan to provide finalized list to W3C Monday-ish, last chance to raise issue of your name is not on it and you believe it should be.
+* AC: each organization has one vote, for example for Inrupt I am the designated voter. You can see Hadrian is not on the list of eligible voters.
+* ML: to be clear, if I am in an organization, I cannot vote?
```

Comment:
@lecoqlibre @andreiciortea , to make sure there is no oversight, e.g., your affiliations not being up to date, may I ask you to both confirm that it is up to date in https://www.w3.org/groups/cg/solid/participants/ , i.e., you're with "Institut National de Recherche en Informatique et en Automatique (Inria)".

If so, great, but also note that as it stands, only @andreiciortea can vote ( as listed inhttps://github.com/w3c-cg/solid/blob/main/elections/2023/eligible-voters.txt ) based on the information we've gathered in https://github.com/w3c-cg/solid/blob/main/elections/2023/affiliation-designated-voters.md and asked the CG to verify in https://lists.w3.org/Archives/Public/public-solid/2023Nov/0075.html

If either of your affiliations should be different, please update that, and let us know about it in an issue at https://github.com/w3c-cg/solid/ . If wish to change the current designated voter from @andreiciortea to @lecoqlibre , you can let us know as well, and perhaps create a PR against https://github.com/w3c-cg/solid/blob/main/elections/2023/affiliation-designated-voters.md .

Reply:
@csarven as Inria AC Rep I confirm the affiliation of @andreiciortea to Inria (and his inria email is used for W3C account) and that he is voting for Inria

Reply:
Great, thanks for confirming Andrei and Fabien. I'm glad this resolves the potential concern. So... happy voting =)